index.html

```diff
@@ -190,14 +187,21 @@ <h3>GitHub page</h3>
         <a href="https://github.com/cmus/cmus">https://github.com/cmus/cmus</a>
       </p>
 
+      <h3>Development</h3>
```

Comment:
The page already has a "Development" header, no need to repeat it. I think these 2 sections can simply be merged.